documentation/src/main/asciidoc/userguide/chapters/query/hql/HQL.adoc

```diff
@@ -414,6 +424,29 @@ include::{sourcedir}/HQLTest.java[tags=hql-api-stream-example]
 Just like with `ScrollableResults`, you should always close a Hibernate `Stream` either explicitly or using a https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html[try-with-resources] block.
 ====
 
+[[jpql-api-stream]]
+==== Query streaming
+
+Since version 2.2, the JPA `Query` interface offers support for returning a `Stream` via the `getResultStream` method.
+
+Just like the `scroll` method, you can use a try-with-resources block to close the `Stream`
+prior to closing the currently running Persistence Context.
+
+Since Hibernate 5.4, the `Stream` is also closed when calling a terminal operation,
```

Comment:
technically speaking, this feature will appear in next version, i.e. 5.4.13, not ever since 5.4 was released.